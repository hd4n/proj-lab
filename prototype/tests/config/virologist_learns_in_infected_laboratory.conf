#25. Virologist learns a code in Infected Laboratory
#@author Eros Pal
#@since 2022-04-23

@items

@effects

@agents

@codes
co c1

@fields
em e1
in i1 c1

@neighbors
e1 i1

@virologists
v1 i1