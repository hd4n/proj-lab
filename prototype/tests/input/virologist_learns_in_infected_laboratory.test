load virologist_learns_in_infected_laboratory.conf
step v1 i1
nextround
interact v1
getstats v1
getstats i1