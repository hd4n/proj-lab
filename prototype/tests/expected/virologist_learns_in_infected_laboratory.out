Load: sikeres
step: v1 i1
nextround: sikeres
interact: v1 i1

stats: v1
	fi_i1
	di_i1
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_-1
	neededNucleo_-1
	co_c1
	ma_null
	ag_null
	eq_null
	eqa_null
	ef_b99

stats: i1
	ci_v1
	ne_e1
	co_c1